cleanup.py

```
@@ -0,0 +1,200 @@
+#!/usr/bin/env python3
+import os
+import sys
+import json
+import subprocess
+import shutil
+import re
+
+def run_command(cmd, cwd=os.getcwd()):
```

Ruff (0.8.2)

Do not perform function call `os.getcwd` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)